xmpp/auth.py

```diff
@@ -65,7 +67,7 @@ def plugin(self,owner):
 
         if query.getTag('digest'):
             self.DEBUG("Performing digest authentication",'ok')
-            query.setTagData('digest',HHSHA1(owner.Dispatcher.Stream._document_attrs['id']+self.password))
+            query.setTagData('digest',sha1((owner.Dispatcher.Stream._document_attrs['id']+self.password).encode('utf-8')).hexdigest())
```

Comment:
Dear @smudge1977,

as you confirmed at https://github.com/xmpppy/xmpppy/issues/38#issuecomment-1065148020, I believe the encoding issue here has been resolved already. If not, please let us know.

With kind regards,
Andreas.